commit 4ea0b438694fd444e4b149faa273bedb58a36b0c
Author: Gökçen Eraslan <gokcen@pardus.org.tr>
Date:   Wed Sep 21 09:09:54 2011 +0000

    Do not set band member by default
    
    If band is set in wireless settings of KNM, it'll always appear in
    connection as band=bg, but its unnecessary. We must set it only when
    user explicitly selects a band value.

diff --git a/libs/internals/settings/802-11-wireless.cpp b/libs/internals/settings/802-11-wireless.cpp
index 2f0dbed..776c5a5 100644
--- a/libs/internals/settings/802-11-wireless.cpp
+++ b/libs/internals/settings/802-11-wireless.cpp
@@ -6,7 +6,7 @@
 using namespace Knm;
 
 WirelessSetting::WirelessSetting() : Setting(Setting::Wireless),
-    mMode(0), mBand(WirelessSetting::EnumBand::bg), mChannel(0), mRate(0), mTxpower(0), mMtu(0)
+    mMode(0), mBand(-1), mChannel(0), mRate(0), mTxpower(0), mMtu(0)
 {
 }
 
